config/settings.json.template

```diff
@@ -11,7 +11,7 @@
       "ttl": 21600000
     },
     "update": {
-      "throttle": 15000
+      "throttle": 5000
```

Comment:
I suggest using 1 second trottle and see how it goes.

This would provide a better user experience.

Reply:
@prlanzarin - what do you think?